lib/galaxy/dependencies/pipfiles/default/pinned-dev-requirements.txt

```
@@ -56,7 +56,7 @@ sphinx-rtd-theme==0.4.3
 sphinx==1.8.5
 sphinxcontrib-websupport==1.1.2
 testfixtures==6.14.0
-twill==0.9.1 ; python_version < '3'
+-e git://github.com/mvdbeek/twill3.git@725889b778a783816a242de3f60f636f7f2dbabf#egg=twill
```

```suggestion
-e git+https://github.com/Cito/twill.git@427f5e647a4279dd5ac5665601ce9e78175c712f#egg=twill
```

Can you also update the Pipfile with:

```diff
diff --git a/lib/galaxy/dependencies/pipfiles/default/Pipfile b/lib/galaxy/dependencies/pipfiles/default/Pipfile
index 5cf77f6156..a11b144c28 100644
--- a/lib/galaxy/dependencies/pipfiles/default/Pipfile
+++ b/lib/galaxy/dependencies/pipfiles/default/Pipfile
@@ -25,7 +25,7 @@ Sphinx = "*"
 sphinx_markdown_tables = "*"
 sphinx_rtd_theme = "*"
 testfixtures = "*"
-twill = {version = "==0.9.1", markers = "python_version < '3'"}
+twill = {git = 'https://github.com/Cito/twill.git', ref = '2.0', editable=true}
 watchdog = "*"
 
 [packages]
```

---

Still needs a minor update for RagioGroup handling in my fork.